src/hyperdrive-cli/commands/stakewise/wallet/claim-rewards.go

```diff
@@ -0,0 +1,28 @@
+package wallet
+
+import (
+	"fmt"
+
+	"github.com/nodeset-org/hyperdrive/hyperdrive-cli/client"
+	"github.com/nodeset-org/hyperdrive/hyperdrive-cli/utils/tx"
+	"github.com/urfave/cli/v2"
+)
+
+func claimRewards(c *cli.Context) error {
+	hd := client.NewHyperdriveClientFromCtx(c)
+	sw := client.NewStakewiseClientFromCtx(c)
+	resp, err := sw.Api.Wallet.ClaimRewards()
+	if err != nil {
+		return err
+	}
+	err = tx.HandleTx(c, hd, resp.Data.TxInfo,
```

Comment:
Not a requirement, just a future note - here is usually where you print info from the response (like amount of ETH available to claim or something) and whether or not claiming is possible. We can add it later.